app/assets/javascripts/discourse/app/components/search-menu/results/types.js

```diff
@@ -22,25 +23,25 @@ export default class Types extends Component {
   }
 
   @action
-  onClick(event) {
+  onClick({ resultType, result }, event) {
     if (wantsNewWindow(event)) {
```

Comment:
so we don't log cmd+clicks?

Reply:
Hmm good point. Not sure how it worked in the past.

Will make it work for all kinds of clicks 👌